### ISSUE TITLE
Feature: Additional cli parameter --exclude

### ISSUE THREAD

**Issue by icanhazstring** [icanhazstring/composer-unused#20]
**Is your feature request related to a problem? Please describe.**
Related to #12 and #11. This should solve problem with false positives where configs and other directories are not parsed using `composer.json` directives only. This was already reworked so should work. The problem with that is, that now there could be some files that should **not** be parsed. The new parameter should be able to exclude a list of folders to ignore.

**Describe the solution you'd like**
Add inputArgument `--exclude` which should be an array, so calls like this can be made:
```
$ composer unused --exclude="vendor" --exclude="legacy"
```

**Comment by icanhazstring** [comment]
Changed this to exclude dirs and packages:
```
$ composer unused --excludeDir="vendor" --excludePackage="fubar"
```

**Comment by icanhazstring** [comment]
Implemented with `0.3.0`
